_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -251,6 +249,29 @@ mfxStatus SetRateControl(
     return MFX_ERR_NONE;
 }
 
+mfxStatus SetBrcResetRequired(
+    VADisplay             vaDisplay,
+    VABufferID &          rateParamBuf_id,
```

Comment:
No need to pass by reference here, pass by value

Reply:
> Good catch. It should be something:
> 
> MFX_CHECK(rateParamBuf_id != VA_INVALID_ID, MFX_ERR_INVALID_HANDLE);

@mgonchar , IMO - no, we shouldn't return an error. If buffer was not created,   just don't do anything here.

Reply:
@dmitryermilov ok, if so, it should be:
```
if (rateParamBuf_id == VA_INVALID_ID)
    return MFX_ERR_NONE;
```

just to avoid putting body of function inside ```if``` statement

Reply:
yes..